.github/workflows/job_onnx_runtime.yml

```diff
@@ -133,7 +95,7 @@ jobs:
         if: ${{ runner.arch != 'ARM64' }} # Ticket: 126277
         run: |
           # see https://github.com/microsoft/onnxruntime/issues/13197#issuecomment-1264542497
-          apt-get install --assume-yes --no-install-recommends language-pack-en
+          apt-get update && apt-get install --assume-yes --no-install-recommends language-pack-en
```

Comment:
why do we need it? I supposed within docker image it's already updated

Reply:
To reduce the image size, the apt lists are deleted after the package installation: 
![image](https://github.com/openvinotoolkit/openvino/assets/65596953/5a9fb5e6-4bd7-4f45-90d4-5c5fe3e4548d)